Jenkinsfile

```diff
@@ -1032,8 +1033,9 @@ pipeline {
             when {
                 beforeAgent true
                 // expression { skipTest != true }
+                expression { env.NO_CI_TESTING != 'true' }
```

Comment:
Probably want to wrap these two expressions in an `allOf { }` wrapper.

Reply:
If there are only conditions, they are treated as if they are contained in an allOf {} wrapper per documentation.